### PR TITLE
hub: update enforce sign-in

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Update enforce sign-in topic based on CSM feedback. See related ticket.

https://deploy-preview-18306--docsdocker.netlify.app/docker-hub/configure-sign-in/
https://deploy-preview-18306--docsdocker.netlify.app/single-sign-on/enforcement-faqs/

### Related issues (optional)

ENGDOCS-1664

### CODE REVIEW

File: content/docker-hub/configure-sign-in.md [hunk 1]
```diff
@@ -5,29 +5,229 @@ keywords: authentication, registry.json, configure,
...
 
-## Deploy registry.json to multiple devices
 
-The previous instructions explain how to create and deploy a registry.json file to a single device. To automatically deploy the registry.json to multiple devices, you must use a third-party solution, such as a mobile device management solution. You can use the previous instructions along with your third-party solution to remotely deploy the registry.json file, or remotely install Docker Desktop with the registry.json file. For more details, see the documentation of your third-party solution.
+## Create a registry.json file to enforce sign-in
+
+1. Ensure that the user is a member of your organization in Docker. For more
+details, see [Manage members](https://docs.docker.com/docker-hub/members/).
+
+2. Create the registry.json file.
```
Comment (stephaurelio) [inline content/docker-hub/configure-sign-in.md:46]: Wondering if all the instances of this should have backticks?
```suggestion
2. Create the `registry.json` file.
```

**Review by stephaurelio — COMMENTED** [review]
This looks great ✨ left a question about if using the backticks around `registry.json` can be more consistent. Otherwise, this LGTM but I'll comment for now in case you also want feedback from Allie!

File: content/docker-hub/configure-sign-in.md [hunk 2]
```diff
@@ -5,29 +5,229 @@ keywords: authentication, registry.json, configure,
...
+    ```
+
+4. Verify that sign-in is enforced.
+
+    Start Docker Desktop on the user’s machine and verify that the **Sign in
+    required!** prompt appears.
+
+## Alternative methods to create a registry.json file
+
+You can also use the following alternative methods to create a `registry.json`` file.
```
Comment (aevesdocker) [inline content/docker-hub/configure-sign-in.md:74]:
```suggestion
You can also use the following alternative methods to create a `registry.json` file.
```

File: content/docker-hub/configure-sign-in.md [hunk 3]
```diff
@@ -5,29 +5,229 @@ keywords: authentication, registry.json, configure,
...
+
+### Download a registry.json file from Docker Hub
+
+In Docker Hub, you can download the `registry.json` file for your organization
+or copy the specific commands to create the file for your organization. To
+download the file or copy the commands, use the following steps.
+
+1. Log in to [Docker Hub](http://hub.docker.com/) as an organization owner.
```
Comment (aevesdocker) [inline content/docker-hub/configure-sign-in.md:82]:
```suggestion
1. Sign in to [Docker Hub](http://hub.docker.com/) as an organization owner.
```

**Review by aevesdocker — COMMENTED** [review]
Nice!

File: content/docker-hub/configure-sign-in.md [hunk 4]
```diff
@@ -5,29 +5,229 @@ keywords: authentication, registry.json, configure,
...
-organization’s settings apply to the user’s session, you can use a `registry.json` file.
+By default, members of your organization can use Docker Desktop without signing
+in. When users don’t sign in as a member of your organization, they don’t
+receive the [benefits of your organization’s
+subscription](../subscription/details.md) and they can circumvent [Docker’s
+security features](../desktop/hardened-desktop/_index.md) for your organization.
 
-The `registry.json` file is a configuration file that allows administrators to specify the Docker organization the user must belong to and ensure that the organization’s settings apply to the user’s session. The Docker Desktop installer can create this file on the users’ machines as part of the installation process.
+To ensure members of your organization always sign in, you can deploy a
+**`registry.json`** configuration file to the machines of your users.
```
Comment (craig-osterhout) [inline content/docker-hub/configure-sign-in.md:15]: Copy-pasted this from notion without noticing that some occurrences got bolded. Will fix.
```suggestion
`registry.json` configuration file to the machines of your users.
```

File: content/docker-hub/configure-sign-in.md [hunk 5]
```diff
@@ -5,29 +5,231 @@ keywords: authentication, registry.json, configure,
...
+
+    | Platform | Location |
+    | --- | --- |
+    | Windows | /ProgramData/DockerDesktop/registry.json |
+    | Mac | /Library/Application Support/com.docker.docker/registry.json |
+    | Linux | /usr/share/docker-desktop/registry/registry.json |
+
+3. Specify your organization in the `registry.json` file.
+
+    Open the `registry.json` file in a text editor and add the following contents, where `myorg`` is replaced with your organization’s name. The file contents are case-sensitive and you must use lowercase letters for your organization's name.
```
Comment (stephaurelio) [inline content/docker-hub/configure-sign-in.md:58]: An extra backtick snuck in: 
```suggestion
    Open the `registry.json` file in a text editor and add the following contents, where `myorg` is replaced with your organization’s name. The file contents are case-sensitive and you must use lowercase letters for your organization's name.
```

**Review by stephaurelio — COMMENTED** [review]
One tiny comment ❇️

**Review by stephaurelio — APPROVED** [review]
LGTM

**Review by sky41906 — COMMENTED** [review]
@sky41906

**Pull request merged**